mitodl/open-discussions
# Add UI to edit post types

#### What are the relevant tickets?
Fixes #831 

#### What's this PR do?
Adds checkboxes selecting the post types allowed (either text, link, or both). 

#### How should this be manually tested?
Go to the edit channel page. Under the Basic tab uncheck all post types then click 'Save'. You should see an error message. Check one of the boxes and you should be directed to the channel page. If you go back to edit it again you should see that the checkbox you selected is still selected.
